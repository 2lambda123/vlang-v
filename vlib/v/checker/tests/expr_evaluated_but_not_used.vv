fn main() {
	{1, 2, 3, 4}
	println('works')
}
