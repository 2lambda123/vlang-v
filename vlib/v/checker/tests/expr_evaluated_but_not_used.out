vlib/v/checker/tests/expr_evaluated_but_not_used.vv:2:3: error: expression evaluated but not used
    1 | fn main() {
    2 |     {1, 2, 3, 4}
      |      ^
    3 |     println('works')
    4 | }
